community_tutorials_and_guides/rf_demo.ipynb

```diff
@@ -88,10 +88,10 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "data_dir = '../../data/rf/'\n",
+    "data_dir = './rf/'\n",
```

Comment:
Curious about why you changed the path?

Reply:
@Nicholas-7, i think we should change the download folder back to the absolute data folder

Reply:
> Using Stable - 0.18 these two notebooks do not work.
> _Utilizing an AWS T4 g4dn.xlarge instance_
> Rf_demo:
> Kernel Dies when trying to train sklearn RF parallely.
> 
> ```
> skl_rf = sklRF(**skl_rf_params, n_jobs=20)
> skl_rf.fit(X_train, y_train)
> ```
> 
> Weather:
> The get_date_parts helper function will not allow for an .astype conversion of an object to an int datatype.
> 
> ```
> Metadata inference failed in `get_date_parts`
> Original error is below:
> ------------------------
> ValueError('Could not convert strings to integer type due to presence of non-integer values.')
> ```

also, i fixed the weather folder issue.

Reply:
Exactly, the file path has been update to point to where the data is stored,